src/picker/rofimoji.py

```diff
@@ -448,26 +452,28 @@ def append_to_favorites_file(self, characters: str) -> None:
             file.write(characters + '\n')
 
     def execute_action(self, characters: str) -> None:
-        if self.args.action == Action.TYPE:
-            self.typer.type_characters(characters, self.active_window)
-        elif self.args.action == Action.COPY:
-            self.clipboarder.copy_characters_to_clipboard(characters)
-        elif self.args.action == Action.CLIPBOARD:
-            self.clipboarder.copy_paste_characters(characters, self.active_window, self.typer)
-        elif self.args.action == Action.UNICODE:
-            self.typer.type_characters(self.get_codepoints(characters), self.active_window)
-        elif self.args.action == Action.COPY_UNICODE:
-            self.clipboarder.copy_characters_to_clipboard(self.get_codepoints(characters))
-        elif self.args.action == Action.STDOUT:
-            print(characters)
+        for action in self.args.actions:
+            if action == Action.TYPE:
+                self.typer.type_characters(characters, self.active_window)
+            elif action == Action.COPY:
+                self.clipboarder.copy_characters_to_clipboard(characters)
+            elif action == Action.CLIPBOARD:
+                self.clipboarder.copy_paste_characters(characters, self.active_window, self.typer)
+            elif action == Action.UNICODE:
+                self.typer.type_characters(self.get_codepoints(characters), self.active_window)
+            elif action == Action.COPY_UNICODE:
+                self.clipboarder.copy_characters_to_clipboard(self.get_codepoints(characters))
+            elif action == Action.STDOUT:
+                print(characters)
 
     def save_selection_to_cache(self, characters: str, processed_characters: str) -> None:
         with cache_file_location.open('w+') as file:
-            file.write(f'{self.args.action.value}\n{characters}\n{processed_characters}')
+            actions = ",".join(action.val for action in self.args.actions)
+            file.write(f'{actions}\n{characters}\n{processed_characters}')
 
     def load_selection_from_cache(self) -> Tuple[str, str]:
         cache = cache_file_location.read_text().split('\n')
-        self.args.action = next(action for action in Action if action.value == cache[0].strip())
+        self.args.actions = [Action(a) for a in cache[0].strip().split(",")]
```

Comment:
I believe this should be backward compatible with previous cache files since `"type".split(",")` (for example) just yields `["type"]`. Also, `Action(a)` should be equivalent to `next(action for action in Action if action.value == a)`.